src/components/Collection/Filter/FilterSidebar.tsx

```diff
@@ -70,7 +70,7 @@ const FilterSidebar = () => {
   return (
     <ShokoPanel
       title="Filter"
-      className="ml-8 w-full"
+      className="sticky top-24 ml-6 !h-[calc(100vh-18rem)] w-full"
```

Comment:
`!h-[calc(100vh-18rem)]`, this, again, as used in various other places in the codebase, is a hack.
But I will let it go since we do it elsewhere